src/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * dex-contracts
+ *
+ * This NPM package provides smart contract artifacts used in the dFusion
+ * protocol. Additional tools for interacting with the dFusion contracts and
+ * performing migrations are also provided.
+ */
+
+const assert = require("assert")
+const BN = require("bn.js")
+
+const BatchExchange = require("../build/contracts/BatchExchange.json")
+const SnappAuction = require("../build/contracts/SnappAuction.json")
+
+/**
+ * Decodes auction elements returned from `BatchExchange.getEncodedAuctionElements`
+ * which are manually encoded into bytes to get around Solidity ABI limitations.
+ * @param {string} bytes The encoded bytes in hex in the form '0x...'
+ * @return {Object[]} The decoded auction elements
+ */
+function decodeRawAuctionElements(bytes) {
```

Comment:
should this functions live in it's own file, or in a utils/helpers file so `index.js` responsibility is just re-exporting things from the project?

Reply:
Yeah, I think that is a good idea.

Reply:
I also renamed the function as it is used for both `getEncodedAcutionElements` and `getEncodedUserOrders`.